src/app/zap-templates/common/ChipTypesHelper.js

```diff
@@ -47,11 +47,18 @@ function asBasicType(type)
 const signedTypes = [ 'INT8S', 'INT16S', 'INT32S', 'INT64S' ];
 function isSigned(type)
 {
-  return signedTypes.includes(type);
+  return signedTypes.includes(type.toUpperCase());
+}
+
+const unsupportedTypes = [ 'INT24S', 'INT40S', 'INT48S', 'INT56S', 'INT24U', 'INT40U', 'INT48U', 'INT56U' ];
+function isUnsupportedType(type)
```

Comment:
Could you add a comment? Unsuppored by what?

I see the usage in simple accessors. Should these be 'canHaveSimpleAccessors' or 'mapstorawcpptype' or something similar?

Reply:
I have opened #8202 and I have created this `common/attributes/Accessors.js` to hold `canHaveSimpleAccessors` and  `isUnsupportedTypes`. That should makes the relationship clearer between those 2 methods.